### PR TITLE
Merge pull request #27 from emerson-eps/dependabot/npm_and_yarn/browserify-sign-4.2.2

### PR DESCRIPTION
**Pull request by syss69**
Images added

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -5,16 +5,34 @@ function App() {
...
-        {
-          /*
-            Please add your custom component(s) here
-          */
-        }
+        <br/>
+        <br/>
+      </div>
+      <div>
+        <img src="https://api.dicebear.com/7.x/fun-emoji/svg?seed=Ginger" height="auto" width={150} alt="Smile" id='smile' />
```
Comment (clairecassan) [inline src/App.tsx:17]: Next task can be to randomize the placement. We will add it to the sprint tomorrow

File: src/App.test.tsx [hunk 2]
```diff
@@ -4,6 +4,6 @@ import App from './App';
 
 test('renders learn react link', () => {
   render(<App />);
-  const header = screen.getByRole("heading", {level: 1})
-  expect(header).toBeInTheDocument();
+  const title = screen.getByRole("title")
+  expect(title).toBeInTheDocument(<p role="title">Elephant game</p>);
 });
```
Comment (clairecassan) [inline src/App.test.tsx:9]: You are missing a test for the "image" feature that you implemented

**Review by clairecassan — CHANGES_REQUESTED** [review]

**Review by clairecassan — APPROVED** [review]

File: src/App.css [hunk 3]
```diff
@@ -13,6 +13,10 @@
...
 }
 
-  .App-img {
-    border:5px solid black
-}
\ No newline at end of file
+  img {
+    border:5px solid black;
+    margin: 1rem;
+    backface-visibility: visible;
```
Comment (clairecassan) [inline src/App.css:19]: unused: please remove

**Review by clairecassan — APPROVED** [review]

**Comment by syss69** [comment]
Why do I need to remove it?

пн, 15 янв. 2024 г. в 15:12, clairecassan ***@***.***>:

**Pull request merged**